assets/css/main.css

```diff
@@ -568,17 +568,63 @@ td.submissions-dropdown {
 .form-inline .form-group div {
     display: inline-block;
 }
-.correct-option {
-  margin-left: -38px;
-  margin-right: 20px;
-  color: green;
+
+/* Mooc-grader questionnaire */
+
+.exercise .aplus-quiz1 .question-area {
+    margin-top: 20px;
+    margin-bottom: 20px;
 }
-.neutral-option {
-  margin-left: -40px;
-  margin-right: 18px;
-  font-size: 18px;
-  color: #f5bc09;
+
+.exercise .aplus-quiz1 .question-description {
+    margin-bottom: 5px;
+}
+
+.exercise .aplus-quiz1 .error-text {
+    margin-top: 5px;
+    margin-bottom: 5px;
+    color: #a94442;
+    font-weight: bold;
+    margin-bottom: 0px;
+}
+
+.exercise .aplus-quiz1 .has-error .radio,
+.exercise .aplus-quiz1 .has-error .checkbox {
+    color: #333;
+}
+
+.exercise .aplus-quiz1 .option-checked {
+    font-weight: bold;
 }
+
+.exercise .aplus-quiz1 .hint {
```

Comment:
These are really good names. However, could the `hint` be more precise like `question-hint` or such?

Reply:
Sure, I can change that!